### ISSUE TITLE
read_int64/write_int64 not available on JRuby

### ISSUE THREAD

**Issue by ankane** [ffi/ffi#717]
Hi, thanks for this gem! In JRuby, it looks like the `write_int64` and `read_int64` methods are missing, but `write_array_of_int64` and `read_array_of_int64` are available. Same situation with `uint` as well.

```ruby
ptr = ::FFI::MemoryPointer.new(:int64)

# both missing
p ptr.write_int64(123)
p ptr.read_int64

# both succeed
p ptr.write_array_of_int64([123])
p ptr.read_array_of_int64(1)
```

FFI version: 1.11.1 (java)
JRuby version: 9.2.7.0
Platform: Mac 10.14

**Comment by ankane** [comment]
It looks like the `read` method is missing as well:

```ruby
ptr.read(:size_t)
```

**Comment by headius** [comment]
This will be fixed by jruby/jruby#5947.

**Comment by larskanis** [comment]
This will be fixed in JRuby-9.3.

**Comment by ankane** [comment]
Great, thanks!
